### PR TITLE
style: input stlying and theming

### PR DESCRIPTION
**Pull request by TeddyAndersson**
The changes made in this pull request affects styling of the input component. The applied styling is the addition of a colorSchema property for reading colors from the theme context (blue, red, purple, green).

### CODE REVIEW

**Review by zilaei — APPROVED** [review]

**Review by dannil76 — APPROVED** [review]
Works on my environment. The code is clean and structured in a good way.

**Comment by zilaei** [comment]
hacktober-fest accepted label?

**Comment by dannil76** [comment]
See my atempt to fix the conflict

File: source/components/atoms/Input/Input.js [hunk 1]
```diff
@@ -9,8 +9,9 @@ const input = css`
   background-color: ${({ theme, colorSchema }) => theme.colors.complementary[colorSchema][2]};
   border-radius: 4.5px;
   border: solid 1px
-    ${({ theme, error, colorSchema }) =>
-      error ? theme.colors.primary.red[0] : theme.colors.complementary[colorSchema][2]};
+  ${({ color, error, colorSchema }) => (error?.isValid || error === undefined)
```
Comment (TeddyAndersson) [inline source/components/atoms/Input/Input.js:12]: Missing theme prop

**Pull request merged**